lib/matplotlib/pyplot.py

```diff
@@ -706,11 +706,11 @@ def xkcd(
     scale: float = 1, length: float = 100, randomness: float = 2
 ) -> ExitStack:
     """
-    Turn on `xkcd <https://xkcd.com/>`_ sketch-style drawing mode.  This will
-    only have effect on things drawn after this function is called.
+    Turn on `xkcd <https://xkcd.com/>`_ sketch-style drawing mode. This will
+    only have an effect on things drawn after this function is called.
 
-    For best results, the "Humor Sans" font should be installed: it is
-    not included with Matplotlib.
+    For best results, install the `xkcd script <https://github.com/ipython/xkcd-font/>`_
```

Comment:
This doesn't seem to be the same link as what is used for CI, and the dependencies page.